src/test/java/com/eleks/academy/whoami/service/GameServiceImplTest.java

```diff
@@ -148,28 +150,41 @@ void enrollToGameTest() {
 	}
 
 	@Test
-	void suggestCharacterTest() {
+	void suggestCharacterWhenGameIsNotFoundTest() {
 		final String player = "Player";
 		CharacterSuggestion suggestion = new CharacterSuggestion("Bet Monkey");
-		final String suggested = "Bet Monkey";
-		ChoosingCharacter character = ChoosingCharacter.builder()
-				.character(suggested)
-				.build();
 
-		SynchronousGame game = new PersistentGame(player, gameRequest.getMaxPlayers());
-		Optional<SynchronousGame> createdGame = Optional.of(game);
-		final String id = game.getId();
-		when(gameRepository.findById(id)).thenReturn(createdGame);
+		SynchronousGame testedGame = new PersistentGame(player, gameRequest.getMaxPlayers());
+		final String id = testedGame.getId();
+
+		assertThrows(ResponseStatusException.class, () -> gameRepository.findById(id)
```

Comment:
this exception should be thrown from the service when you call it, here as I see that code from the service is taken and put here, so in this case, my question is - what do we test?